params/safety_config.yaml

```diff
@@ -0,0 +1,11 @@
+safety_bonds:
```

Comment:
It looks like if you move the call to main.launch to the top of the abstract files and the rosparam of safety_config.yaml to the top of base hardware you'll be able to load the safety_config.yaml before everything else.